{"primaryContentSections":[{"kind":"declarations","declarations":[{"tokens":[{"kind":"keyword","text":"typealias"},{"kind":"text","text":" "},{"kind":"identifier","text":"Failure"},{"kind":"text","text":" = "},{"kind":"typeIdentifier","identifier":"doc:\/\/com.processout.processout-ios\/documentation\/ProcessOut\/POFailure","preciseIdentifier":"s:10ProcessOut9POFailureV","text":"POFailure"}],"languages":["swift"],"platforms":["iOS"]}]}],"schemaVersion":{"major":0,"minor":3,"patch":0},"sections":[],"variants":[{"paths":["\/documentation\/processout\/porepository\/failure"],"traits":[{"interfaceLanguage":"swift"}]}],"identifier":{"url":"doc:\/\/com.processout.processout-ios\/documentation\/ProcessOut\/PORepository\/Failure","interfaceLanguage":"swift"},"abstract":[{"type":"text","text":"Repository’s failure type."}],"kind":"symbol","metadata":{"fragments":[{"kind":"keyword","text":"typealias"},{"kind":"text","text":" "},{"kind":"identifier","text":"Failure"}],"title":"PORepository.Failure","roleHeading":"Type Alias","role":"symbol","symbolKind":"typealias","externalID":"s:10ProcessOut12PORepositoryP7Failurea","modules":[{"name":"ProcessOut"}],"navigatorTitle":[{"kind":"identifier","text":"Failure"}]},"hierarchy":{"paths":[["doc:\/\/com.processout.processout-ios\/documentation\/ProcessOut","doc:\/\/com.processout.processout-ios\/documentation\/ProcessOut\/PORepository"]]},"references":{"doc://com.processout.processout-ios/documentation/ProcessOut/PORepository":{"role":"symbol","title":"PORepository","fragments":[{"kind":"keyword","text":"protocol"},{"kind":"text","text":" "},{"kind":"identifier","text":"PORepository"}],"abstract":[{"type":"text","text":"Common protocol that all repositories conform to."}],"identifier":"doc:\/\/com.processout.processout-ios\/documentation\/ProcessOut\/PORepository","kind":"symbol","type":"topic","navigatorTitle":[{"kind":"identifier","text":"PORepository"}],"url":"\/documentation\/processout\/porepository"},"doc://com.processout.processout-ios/documentation/ProcessOut":{"role":"collection","title":"ProcessOut","abstract":[],"identifier":"doc:\/\/com.processout.processout-ios\/documentation\/ProcessOut","kind":"symbol","type":"topic","url":"\/documentation\/processout"},"doc://com.processout.processout-ios/documentation/ProcessOut/PORepository/Failure":{"role":"symbol","title":"PORepository.Failure","fragments":[{"kind":"keyword","text":"typealias"},{"kind":"text","text":" "},{"kind":"identifier","text":"Failure"}],"abstract":[{"type":"text","text":"Repository’s failure type."}],"identifier":"doc:\/\/com.processout.processout-ios\/documentation\/ProcessOut\/PORepository\/Failure","kind":"symbol","type":"topic","navigatorTitle":[{"kind":"identifier","text":"Failure"}],"url":"\/documentation\/processout\/porepository\/failure"},"doc://com.processout.processout-ios/documentation/ProcessOut/POFailure":{"role":"symbol","title":"POFailure","fragments":[{"kind":"keyword","text":"struct"},{"kind":"text","text":" "},{"kind":"identifier","text":"POFailure"}],"abstract":[{"type":"text","text":"Information about an error that occurred."}],"identifier":"doc:\/\/com.processout.processout-ios\/documentation\/ProcessOut\/POFailure","kind":"symbol","type":"topic","navigatorTitle":[{"kind":"identifier","text":"POFailure"}],"url":"\/documentation\/processout\/pofailure"}}}